build.sbt

```diff
@@ -7,9 +7,8 @@ lazy val root = (project in file(".")).enablePlugins(PlayJava)
 scalaVersion := "2.11.8"
 
 libraryDependencies += javaJpa
-
-libraryDependencies += "org.mockito" % "mockito-core" % "2.1.0"
+libraryDependencies += "org.hibernate" % "hibernate-core" % "5.2.5.Final"
 
 libraryDependencies += javaWs % "test"
-
-libraryDependencies += "org.hibernate" % "hibernate-core" % "5.2.5.Final"
+libraryDependencies += "org.mockito" % "mockito-core" % "2.1.0" % "test"
+testOptions in Test += Tests.Argument(TestFrameworks.JUnit, "-a", "-v")
```

Comment:
These are the defaults in junit-interface master. If you can get a new release cut then we don't need this line